vars/database.yml

```diff
@@ -0,0 +1,38 @@
+---
+database_host: localhost
+database_port: 5432
+database_ssl: false
+database_ssl_mode: disable
+database_ssl_ca: None
+
+foreman_database_name: foreman
+foreman_database_user: foreman
+foreman_database_password: CHANGEME
+candlepin_database_user: candlepin
+candlepin_database_name: candlepin
+candlepin_database_password: CHANGEME
+pulp_database_user: pulp
+pulp_database_name: pulp
+pulp_database_password: CHANGEME
```

Comment:
For these I think we have two options. 

Option 1: These serve as the role variables and we expose them externally to the user as the inputs.

Option 2: These server as the external inputs for the user and those map to internal role variables (e.g. foreman_db_name)

Reply:
> Option 1: These serve as the role variables and we expose them externally to the user as the inputs.

I like this best.

I also lean to `X_database_Y` if you also have `database_Y`. More concrete: `--foreman-database-password` and `--database-host`. I'll also accept `--foreman-db-password` and `--db-host` but think writing out `database` isn't that bad.

Random thought: do we want to accept aliases? Probably not, but this is a good example where it could make sense.

Reply:
> Random thought: do we want to accept aliases? Probably not, but this is a good example where it could make sense.

Can you define aliases in this context?